sphinx/locale/__init__.py

```diff
@@ -255,17 +255,18 @@ def get_translation(catalog, namespace='general'):
         import os
         from sphinx.locale import get_translation
 
-        _ = get_translation(__name__)
+        EXTENSION_NAME = 'myextension'  # name of both *.pot, *.po and *.mo files
```

Comment:
We usually call these files as "message catalog".

If you meant this is the name of message catalog, it would be better to rename the variable name to `MESSAGE_CATALOG_NAME`.